digdag-server/src/main/java/io/digdag/server/rs/AttemptResource.java

```diff
@@ -302,7 +302,19 @@ public Response startAttempt(RestSessionAttemptRequest request)
                 .getSessionStore(getSiteId())
                 .getTasksOfAttempt(attemptId);
 
+        // Collect tasksIds in which a group retry occurred.
+        // Need to filter the status of the task with "group_error" and "retry_count" in stateParams.
```

Comment:
>         // Need to filter the status of the task with "group_error" and "retry_count" in stateParams.

This might be a silly question... but what does it mean if the group error task has no `retry_count`?

Reply:
It means that group error has occurred, but no group retry has occurred. I added the same description to the codebase :)